src/components/Main/Modal/GroupModal/GroupModal.vue

```diff
@@ -36,7 +36,10 @@ export default defineComponent({
     const group = computed(() => store.state.entities.userGroups[props.groupId])
     const groupName = computed(() => group.value?.name)
     const groupMember = computed(
-      () => group.value?.members.map(member => member.id) ?? []
+      () =>
+        group.value?.members
+          .map(member => member.id)
+          .filter(id => store.state.entities.users[id]) ?? []
```

Comment:
`store.state.entities.users`には取得されてるデータが入ってるんですが、suspendedなユーザーも入り得るので下みたいな感じでお願いしたいです:pray:

```suggestion
          .filter(user => user.state === UserAccountState.active)
          .map(member => member.id) ?? []
```

Reply:
ありがとうございます :pray:　修正しました

`group.value?.members` にはidとroleしか入っていなかったので 
```js
          .filter(
            member =>
              store.state.entities.users[member.id]?.state ===
              UserAccountState.active
          )
          .map(member => member.id) ?? []
```
にしました